client/store-management-links/quick-link/test/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React from '@wordpress/element';
+import { render } from '@testing-library/react';
+import { brush } from '@wordpress/icons';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+
+/**
+ * Internal dependencies
+ */
+import { QuickLink } from '../index';
+
+describe( 'QuickLink', () => {
+	it( 'renders an title, href and icon based on props passed', () => {
```

Comment:
Is this missing the assertion for the icon?

Reply:
Oh yes it is, I struggled with doing assertions on the icon svg for some reason. I'll remove that from the assertion

Reply:
Removed the assertion. I still haven't had luck querying svgs by role or any other attribute in jest-dom.